### DISCUSSION TITLE
is it possible to link cosmos with c?

### DISCUSSION THREAD

**Discussion by AsertCreator** [CosmosOS/Cosmos#2621] [Q&A] [answered]
cosmos is compiled to elf object file which is then linked to executable file. theoretically, it is possible to link cosmos with some more object files, compiled from c for example. will it interfere with some cosmos structures and will it be implemented?

**Comment by zarlo** [comment, marked as answer]
it could be done but will not like be supported (you can do it in "userland" but)

**Reply by AsertCreator** [reply]
ok
